### ISSUE TITLE
hey bro is the daily limit around 2000 coin??

### ISSUE THREAD

**Issue by Ctmax-ui** [Lomusire/unlimited-cloud-storage#10]

**Comment by Lomusire** [comment]
hm shouldn't be like that after it finished miner game it should move on to gemmerge but gemmerge is realy slow and only credits the coins every 100 leels and curently extension is prety unstable and sometimes is stoped from running in background due to manifet v3

**Comment by Lomusire** [comment]
so you won't see coins updating until it finished level 100 in gemmerge

**Comment by Lomusire** [comment]
but as you probably alredy saw i had to make the extension ultra slow in order to "bypass" anticheat and gemmerge requests simply fail if you try to send it more quickly

**Comment by Ctmax-ui** [comment]
hmmm that's an big issue, I should try if I can somehow trick the system that's will be an huge upgrades to the extension. how do u get the mobile app api endpoints of the games so I can check the logic ??

**Comment by Ctmax-ui** [comment]
do u have some kind of route lists of something like that??

**Comment by Lomusire** [comment]
i just monitor the network trafic of the terabox app using http toolkit and a tooted device (emulator would work tho)

since for terabox requests aren't encrypted http toolkit work fine for other apps you ca use burpsuite with android crypto interceptor to decrypt the requests https://codeshare.frida.re/@hyugogirubato/android-crypto-interceptor/

**Comment by Lomusire** [comment]
i don't think you can trick the system anymore since reducing the delays triggers anticheat and leads to blacklist of miner game and as its an api its handeld all serverside so you can't just send shorter times anymore and they also decrased limit of playable miner game per day so i added gemmerge wich is way slower than miner but has higer limits

**Comment by Ctmax-ui** [comment]
ok thx, do u have any other means of communication like u see I am creating an git issue to connect with you,
 and also I will look into the api tomorrow, currently I am working on the in extension purchase so you do not have to go through 
![image](https://github.com/user-attachments/assets/fc90fedc-ae99-461f-9f65-158257d07239)
this again an again

**Comment by trungking** [comment]
> i don't think you can trick the system anymore since reducing the delays triggers anticheat and leads to blacklist of miner game and as its an api its handeld all serverside so you can't just send shorter times anymore and they also decrased limit of playable miner game per day so i added gemmerge wich is way slower than miner but has higer limits

gem merger is currently stop after each 100 level and i have to click start again, can you make it auto repeat?

**Comment by Lomusire** [comment]
oh realy? sory didn't realise i forgot to loop it

**Comment by Lomusire** [comment]
you can contact me on discord @lomusire

**Comment by Ctmax-ui** [comment]
![image](https://github.com/user-attachments/assets/57df62b9-2631-402d-ad24-aa7f51b6b2ee)


> you can contact me on discord @Lomusire

**Comment by Ctmax-ui** [comment]
send u an fr

**Comment by shivasharma63420** [comment]
I not getting single coin plz help me

**Comment by Ctmax-ui** [comment]
> I not getting single coin plz help me

Hello user @shivasharma63420, can you describe the issue you are facing properly with reference image and some details so we can know the real problem.

**Comment by shivasharma63420** [comment]
![image](https://github.com/user-attachments/assets/6263ab39-6caf-47ab-a082-588e2def1d33). Fetch failed issue i am getting all time and I received 1 coin only

**Comment by Ctmax-ui** [comment]
Okay, I see. How many coins did you collect before this issue occurred? I’m experiencing a problem where, after collecting around 2,000 coins, it runs but doesn’t give any rewards. If you're facing the same issue, we're currently working on a solution and hope to resolve it as soon as possible. Thank you for reaching out.

**Comment by shivasharma63420** [comment]
I got 1274coins when I restarted then we got only 1 coin in 1 time

**Comment by shivasharma63420** [comment]
Plz add option which game we need to select

**Comment by Ctmax-ui** [comment]
ok we will look onto that

> I got 1274coins when I restarted then we got only 1 coin in 1 time
hmm look like the same issue.

> Plz add option which game we need to select
yes we plan on doing that.
